apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressBalancesAlph.ts

```diff
@@ -19,27 +19,17 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 import { useQuery } from '@tanstack/react-query'
 
 import { UseFetchAddressProps } from '@/api/apiDataHooks/address/addressApiDataHooksTypes'
-import useFetchAddressUpdatesSignal from '@/api/apiDataHooks/address/useFetchAddressUpdatesSignal'
 import { addressAlphBalancesQuery } from '@/api/queries/addressQueries'
 import { useAppSelector } from '@/hooks/redux'
 
 const useFetchAddressBalancesAlph = ({ addressHash, skip }: UseFetchAddressProps) => {
   const networkId = useAppSelector((s) => s.network.settings.networkId)
 
-  const { data: updatesSignal, isLoading: isLoadingUpdatesSignal } = useFetchAddressUpdatesSignal({ addressHash, skip })
-  const { data, isLoading: isLoadingAlphBalances } = useQuery(
-    addressAlphBalancesQuery({
-      addressHash,
-      networkId,
-      latestTxHash: updatesSignal?.latestTx?.hash,
-      previousTxHash: updatesSignal?.previousTx?.hash,
-      skip: skip || isLoadingUpdatesSignal
-    })
-  )
```

Comment:
Without the need for including the `latestTxHash` in the `queryKey` and the `previousTxHash` for data prefilling, our hooks become back cleaner 🫧